api/src/pcapi/core/finance/api.py

```diff
@@ -1817,17 +1830,15 @@ def find_reimbursement_rule(rule_reference: str | int) -> models.ReimbursementRu
 
 
 def _make_invoice_line(
-    group: models.RuleGroup,
-    pricings: list,
-    line_rate: decimal.Decimal | None = None,
+    group: models.RuleGroup, pricings: list, line_rate: decimal.Decimal | None = None, is_incident_line: bool = False
 ) -> tuple[models.InvoiceLine, int]:
     reimbursed_amount = 0
     flat_lines = list(itertools.chain.from_iterable(pricing.lines for pricing in pricings))
-    # positive
+    # ingoing
     contribution_amount = sum(
         line.amount for line in flat_lines if line.category == models.PricingLineCategory.OFFERER_CONTRIBUTION
     )
-    # negative
+    # outgoing
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
** ❓ question:** est-ce qu'il est indiqué quelque part le signe mathématique de ingoing/outgoing ? ‏

Reply:
Oui : 

https://github.com/pass-culture/pass-culture-main/blob/ab48272df3a7d7be422dbe7a1ad7e967f8d5ba44/api/src/pcapi/core/finance/models.py#L1-L7